src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/EvidenceUploadApplicantNotificationHandler.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.notification;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.config.properties.notification.NotificationsProperties;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.NotificationService;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.NOTIFY_APPLICANT_SOLICITOR_FOR_EVIDENCE_UPLOAD;
+
+@Service
+@RequiredArgsConstructor
+public class EvidenceUploadApplicantNotificationHandler extends CallbackHandler implements NotificationData {
+
+    private static final List<CaseEvent> EVENTS = List.of(NOTIFY_APPLICANT_SOLICITOR_FOR_EVIDENCE_UPLOAD);
+
+    private static final String REFERENCE_TEMPLATE = "evidence-upload-notification-%s";
+    public static final String TASK_ID = "EvidenceUploadNotifyApplicantSolicitor";
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
+            callbackKey(ABOUT_TO_SUBMIT), this::notifyApplicantEvidenceUpload
+        );
+    }
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
```

Comment:
Let's add unit-tests around these ones too.

They may be very straightforward, but errors here can prevent things from working correctly so better have dedicated tests.

Plus it would increase test coverage to 100%.